pysmt/walkers/__init__.py

```diff
@@ -33,7 +33,7 @@
 
 """
 
-__all__ = ["DagWalker", "TreeWalker", "IdentityDagWalker"]
+__all__ = ["DagWalker", "TreeWalker", "IdentityDagWalker", "handles"]
```

Comment:
I think this is a left-over of when all walkers were defined in a single file. Ok with removing it.